modules/core/include/visp3/core/vpCameraParameters.h

```diff
@@ -351,10 +380,12 @@ class VISP_EXPORT vpCameraParameters
   static const double DEFAULT_KDU_PARAMETER;
   static const vpCameraParametersProjType DEFAULT_PROJ_TYPE;
 
-  double px, py; //!< pixel size
-  double u0, v0; //!<  principal point
-  double kud;    //!< radial distortion (from undistorted to distorted)
-  double kdu;    //!< radial distortion (from distorted to undistorted)
+  double px, py;                       //!< Pixel size
+  double u0, v0;                       //!< Principal point
+  double kud;                          //!< Radial distortion (from undistorted to distorted)
+  double kdu;                          //!< Radial distortion (from distorted to undistorted)
+  bool m_isKannalaBrandt;              //!< Flag set if Kannala-Brandt camera model (lens distortion)
```

Comment:
There is the [`vpCameraParametersProjType` enum](https://visp-doc.inria.fr/doxygen/visp-3.3.0/classvpCameraParameters.html#a256a1d8004646b6e22a996a84a42dd2f).

If possible, I think it is better to add this new model into it.

Reply:
I confirm. You could introduce `KannalaBrandtProjWithDistortion`in the enum

Reply:
Done.